### PR TITLE
Add redistribute property to bgp_af

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
Beaker passes, rubocop clean.

### CODE REVIEW

**Comment by glennmatthews** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -417,6 +417,31 @@ Example: IPv6 Networks Array
 ]
 ```
 
+##### `redistribute`
+A list of redistribute directives. Multiple redistribute entries are allowed. The list must be in the form of a nested array: the first entry of each array defines the source-protocol to redistribute from; the second entry defines a route-map/route-policy name. A route-map/route-policy is highly advised but may be optional on some platforms, in which case it may be omitted from the array list.
+
+Example: Platform requiring route-maps
+
+```
```
Comment (mikewiebe) [inline README.md]:
```ruby

File: README.md [hunk 2]
```diff
@@ -417,6 +417,31 @@ Example: IPv6 Networks Array
...
+                 ['lisp',    'rm_lisp'],
+                 ['static',  'rm_static'],
+                 ['eigrp 1', 'rm_eigrp'],
+                 ['isis 2',  'rm_isis'],
+                 ['ospf 3',  'rm_ospf'],
+                 ['rip 4',   'rm_rip']]
+```
+Example: Platform with optional route-maps
+
+```
```
Comment (mikewiebe) [inline README.md]: \```ruby

File: examples/demo_bgp.pp [hunk 3]
```diff
@@ -58,7 +58,8 @@
   # Configure Address Family IPv4 Unicast                                     #
   # --------------------------------------------------------------------------#
   $ipv4_networks = [['192.168.5.0/24', 'nrtemap1'], ['192.168.6.0/32']]
-  #$networks = []
```
Comment (mikewiebe) [inline examples/demo_bgp.pp]: oops :)

**Comment by mikewiebe** [comment]
Just a couple nits but otherwise :+1:

**Pull request merged**